src/data.rs

```diff
@@ -277,6 +277,7 @@ pub struct YamlTask {
     pub build: bool,
     #[serde(default = "default_as_false")]
     pub ignore_ctrl_c: bool,
+    pub working_directory: Option<Utf8PathBuf>,
```

Comment:
This is actually not a (Utf8)PathBuf, as it might contain variables, like `${appdir}`.

Reply:
You mean that it is a `String` at this point?

Reply:
Yes, sorry!